libs/perun/components/src/lib/members-dynamic-list/members-dynamic-list.component.css

```diff
@@ -5,3 +5,11 @@
 .cursor-pointer {
   cursor: pointer;
 }
+
+.cursor-default {
+  cursor: default;
+}
+
+::ng-deep .status-tooltip {
```

Comment:
`::ng-deep` is deprecated, please find a different solution

Reply:
Replaced with 'encapsulation: ViewEncapsulation.None' option